src/OpenTracing/ITracer.cs

```diff
@@ -8,10 +10,17 @@ namespace OpenTracing
     public interface ITracer
     {
         /// <summary>
-        /// Returns a new <see cref="ISpanBuilder" /> for a span with the given <paramref name="operationName" />.
+        /// Starts a new Span
         /// </summary>
         /// <param name="operationName">The operation name of the span.</param>
-        ISpanBuilder BuildSpan(string operationName);
+        ISpan StartSpan(
+            string operationName,
+            IEnumerable<Tuple<string, ISpanContext>> references = null,
```

Comment:
This should have a type, but Reference seemed too close to References. If we accept this, should we rename References to KnownReferences, or make a Reference type and have the constants exposed as static members with that type (seems like violation of SRP).